tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html

```diff
@@ -5032,10 +5036,39 @@ <h2>Show similarity to selected datapoint</h2>
        */
       addChart: function(chartType, featureName, data){
         let chart;
+        let dataToChart = data;
+
+        // If multiclass and there are more inferred classes than the set
+        // number of classes to display, then filter the classes to display
+        // in the chart down to the top scoring classes.
+        if (this.isMultiClass_(this.modelType, this.multiClass)) {
+          dataToChart = [];
+          for (let modelIdx = 0; modelIdx < data.length; modelIdx++) {
+            let modelDataToChart = {};
+            if (Object.keys(data[modelIdx]).length >
+                parseInt(this.maxInferenceEntriesPerRun)) {
```

Comment:
`parseInt` should have a radix or you are supposed to use `Number(this.maxInferenceEntriesPerRun)` unless base is not 10.

Reply:
+1, but stronger: `parseInt` should have a radix, but even then `Number`
is preferred by Google style, because `parseInt("1 2 3")` is `1` while
`Number("1 2 3")` is `NaN`. (Our internal TS style guide says “must not”
use `parseInt`.)

Reply:
done